tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bookmark_menu.py

```diff
@@ -33,6 +33,10 @@ def tap_add_bookmark_to_home_screen_dialog_button(self):
         Wait(self.marionette).until(lambda m: self.apps.displayed_app.name != self.name)
         self.apps.switch_to_displayed_app()
 
+    def is_add_to_home_button_not_in_dialog(self):
```

Comment:
agree with Johan, this method should return True is the option is displayed and False if not.